### ISSUE TITLE
Missing closing parenthesis in JSON profiles

### ISSUE THREAD

**Issue by botherder** [subgraph/oz#6]
It seems that almost all the JSON profiles are corrupted and are missing a closing ```}```.
That is causing errors when launching an oz-setup.

```
$ cat gajim.json  | python -m json.tool
Expecting object: line 35 column 2 (char 812)
```

Or when starting oz-daemon:
```
2015/08/20 17:18:18 Failed to load profiles: error loading 'gajim.json': unexpected end of JSON input
```

**Comment by dma** [comment]
Oops. That's what happens when we make a bunch of commits and then go to camp. Will fix ASAP.

**Comment by dma** [comment]
Fixed in 2daf6ce05374f978b207f09955c2e51d71545708.
